src/pybind/mgr/dashboard/controllers/cluster_configuration.py

```diff
@@ -3,14 +3,34 @@
 
 import cherrypy
 
-from . import ApiController, RESTController
+from . import ApiController, RESTController, ControllerDoc, EndpointDoc
 from .. import mgr
 from ..security import Scope
 from ..services.ceph_service import CephService
 from ..exceptions import DashboardException
 
 
+FILTER_SCHEMA = [{
+    "name": (str, 'Name of the cluster'),
+    "type": (str, 'Cluster type'),
+    "level": (str, 'level of info'),
+    "desc": (str, 'description of the configuration'),
+    "long_desc": (str, 'elaborated description'),
+    "default": (str, 'Default value set for the cluster'),
+    "daemon_default": (str, 'Default Daemon associated to the cluster'),
+    "tags": ([str], 'tags associated with the cluster'),
+    "services": ([str], 'services associated with the cluster'),
+    "see_also": ([str], 'additional info'),
+    "enum_values": ([str], 'List of enums allowed'),
+    "min": (str, 'Minimum values'),
+    "max": (str, 'Maximum values'),
+    "can_update_at_runtime": (bool, 'check if can update at runtime'),
+    "flags": ([str], 'List of flags associated')
```

Comment:
Would it make sense/be possible to add the list of valid flags ['runtime', 'no_mon_update', 'startup', 'cluster_create', 'create'] and a link to the documentation: https://docs.ceph.com/docs/master/dev/config/#flags here?

Reply:
We can add the link to documentation in the description (along with 'List of flags associated'), but that would be shown as a string only(not an active link) or similarly we can add list of valid flags to the description itself.